packages/debug/src/browser/model/debug-thread.tsx

```diff
@@ -230,11 +230,44 @@ export class DebugThread extends DebugThreadData implements TreeElement {
 
     render(): React.ReactNode {
         const reason = this.stoppedDetails && this.stoppedDetails.reason;
-        const status = this.stoppedDetails ? reason ? `Paused on ${reason}` : 'Paused' : 'Running';
-        return <div className='theia-debug-thread' title='Thread'>
-            <span className='label'>{this.raw.name}</span>
-            <span className='status'>{status}</span>
-        </div>;
+        const localizedReason = this.getlocalizedReason(reason);
+
+        const status = this.stoppedDetails
+            ? reason
+                ? nls.localizeByDefault('Paused on {0}', localizedReason)
+                : nls.localizeByDefault('Paused')
+            : nls.localizeByDefault('Running');
+        return (
+            <div className="theia-debug-thread" title="Thread">
+                <span className="label">{this.raw.name}</span>
+                <span className="status">{status}</span>
+            </div>
+        );
+    }
+
+    private getlocalizedReason(reason: string | undefined): string {
+        switch (reason) {
+            case 'step':
+                return nls.localize('theia/debug/step', 'step');
+            case 'breakpoint':
+                return nls.localize('theia/debug/breakpoint', 'breakpoint');
+            case 'breakpoint':
+                return nls.localize('theia/debug/exception', 'exception');
+            case 'breakpoint':
+                return nls.localize('theia/debug/pause', 'pause');
+            case 'breakpoint':
+                return nls.localize('theia/debug/entry', 'entry');
+            case 'breakpoint':
+                return nls.localize('theia/debug/goto', 'goto');
+            case 'breakpoint':
+                return nls.localize('theia/debug/function breakpoint', 'function breakpoint');
```

Comment:
Suggestion: Don't use whitespace in translation keys
```suggestion
                return nls.localize('theia/debug/functionBreakpoint', 'function breakpoint');
```